nixos/tests/ipv6.nix

```
@@ -86,6 +87,14 @@ import ./make-test-python.nix ({ pkgs, lib, ...} : {
           # Default route should have "src <temporary address>" in it
           client.succeed(f"ip r g ::2 | grep {ip}")
 
+      with subtest("disabling privacy extensions by default"):
+          ip = wait_for_address(client, "eth1", "global")
+          # Should be an EUI-64 address; while it's possible for ff:fe
+          # to show up "by chance" in non-EUI-64 addresses too, I
+          # think it's unlikely enough that we can take that chance.
+          assert "ff:fe" in ip
+          print(client.succeed(f"ip r g ::2"))
```

```suggestion
          print(client.succeed("ip r g ::2"))
```
Interpolation is not needed.